gapic/res/text/keyboard-mouse-help.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html>
+<head i18n-values="dir:textdirection;">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<meta charset="utf-8"/>
+<title>Keyboard/Mouse Help</title>
+<body>
+   <style>
+    :host {
+      flex: 1 1 auto;
+      flex-direction: row;
+      display: flex;
+      width: 700px;
+    }
+    .pair {
+      flex: 1 1 auto;
+      flex-direction: row;
+      display: flex;
+    }
+    .column {
+      padding: 0 40px 40px 0;
+      vertical-align: top;
+    }
+    .command {
+      font-family: monospace;
+      display:inline;
+      margin-right: 5px;
+      text-align: right;
+      min-width: 200px;
+    }
+    .action {
+      font-size: 0.8em;
+      display:inline;
+      text-align: left;
+      min-width: 400px;
+    }
+    h4 {
+      font-weight: bold;
+      font-size: 0.9em;
+    }
+
+    </style>
+
+    <table>
+    <tr>
+     
+    <td class="column">
+      <h2>General</h2>
+      <div class="pair">
+        <div class="command">1-4</div>
+        <div class="action">Switch mouse mode</div>
+      </div>
+
+      <div class="pair">
+        <div class="command">shift</div>
+        <div class="action">Hold for temporary select</div>
+      </div>
+
+      <div class="pair">
+        <div class="command">space</div>
+        <div class="action">Hold for temporary pan</div>
+      </div>
+
+      <div class="pair">
+        <div class="command">f</div>
```

Comment:
This doesn't work either. Should it be commented out for now? Are we planning to implement it?

Reply:
Correction: It only works with single selections.

Reply:
For what it's worth: both of the zooming key 'f' and the highlighting key 'm' only works for single selection for now. I think we can still annotate this implemented part. If multi-selection is handled later, this documentation can be updated together then.